xlml/apis/test_config.py

```diff
@@ -183,6 +185,22 @@ def setup_script(self) -> Optional[str]:
   def test_script(self) -> str:
     return '\n'.join(self.run_model_cmds)
 
+  @property
+  def startup_script(self) -> str:
+    if self.use_startup_script == False:
+      return None
+
+    main_command = '\n'.join(self.set_up_cmds + self.run_model_cmds)
```

Comment:
Let's wrap the command in `bash -c` and use an escaped string instead of direct formatting into `(...)`: https://docs.python.org/dev/library/shlex.html#shlex.quote

Reply:
Done.